.github/workflows/pr-title-check.yml

```diff
@@ -9,17 +9,15 @@
 name: Check PR title
```

Comment:
This should be tagged as a chore - it only affect dev process, not the seed for the end user